apps/admin-preset-default/src/components/DefaultShippingMethodsInput.vue

```diff
@@ -59,7 +63,7 @@ import {
   useShippingMethodsInputContext,
 } from '@myparcel-pdk/admin';
 import {isEnumValue} from '@myparcel/ts-utils';
-import {PackageTypeName} from '@myparcel/constants';
+import {PackageTypeName} from '@myparcel/constants'; // eslint-disable-next-line vue/no-unused-properties
```

Comment:
Deze comment lijkt er nu dubbel in te staan.

Reply:
Oh ja, dat doet eslint zelf 🙄 ik haal 'm weg